crates/tests-e2e/tests/e2e_collaborative_close_channel.rs

```diff
@@ -43,6 +43,7 @@ async fn can_open_and_collab_close_channel() {
 
     // wait until there is no balance off-chain anymore
     wait_until!({
+        test.bitcoind.mine(1).await.unwrap();
```

Comment:
🔧 Not wrong, but shouldn't it be enough to mine 1 block i.e. just before the `wait_until`?

Reply:
Potentially, 1 block should be enough.